lib/index.js

```diff
@@ -71,6 +71,9 @@ export default function runContentfulExport (usageParams) {
           }
         )
       }
+      return response
```

Comment:
wow, could you flip the `if (opts.downloadAssets) {` on top to `if (!opts.downloadAssets) { return response }`. There's a lot stuff in between and it's difficult to follow the code.